test/lib/mayaUsd/render/mayaToHydra/testDataProducerSelHighlight.py

```diff
@@ -0,0 +1,186 @@
+        cmds.modelEditor(panel, edit=True, displayAppearance="wireframe")
+        self.assertSnapshotClose("Storm_Wireframe_AllSelected.png", self.IMAGE_DIFF_FAIL_THRESHOLD, self.IMAGE_DIFF_FAIL_PERCENT)
+
+        #Switch to bounding box display mode, we should keep the selected items and their color
+        cmds.modelEditor(panel, edit=True, displayAppearance="boundingBox")
+        self.assertSnapshotClose("Storm_BoundingBox_AllSelected.png", self.IMAGE_DIFF_FAIL_THRESHOLD, self.IMAGE_DIFF_FAIL_PERCENT)
+
+        #Switch to wireframe on shaded display mode, we should keep the selected items and their color
+        cmds.modelEditor(panel, edit=True, displayAppearance="smoothShaded")
+        cmds.modelEditor(panel, edit=True, wireframeOnShaded=True)
+        self.assertSnapshotClose("Storm_WireOnShaded_AllSelected.png", self.IMAGE_DIFF_FAIL_THRESHOLD, self.IMAGE_DIFF_FAIL_PERCENT)
+
+    def test_LeadAndActiveColorsSelectionHighlighting(self):
+        import usdUtils # usdUtils imports mayaUsd.ufe
+        from mayaUsd import lib as mayaUsdLib
+
+        #Load the scene        
+        testFile = mayaUtils.openTestScene(
+                "testWireframeSelectionHighlight",
+                "testSelectionHighlightMayaUsd.ma")
+        
+        #Switch to HdStorm
+        self.setHdStormRenderer()
+        cmds.refresh()
+
+        #Prepare for selecting usd prims through ufe
+        shapeNode = "sample_usdShape"
+        shapeStage = mayaUsdLib.GetPrim(shapeNode).GetStage()
+        
+        #define ufe items
+        sphere1Prim = shapeStage.GetPrimAtPath('/sample_usdShape/pSphere1')
+        self.assertIsNotNone(sphere1Prim)
+        sphere2Prim = shapeStage.GetPrimAtPath('/sample_usdShape/pSphere2')
+        self.assertIsNotNone(sphere2Prim)
+        torusPrim = shapeStage.GetPrimAtPath('/sample_usdShape/pTorus1')
+        self.assertIsNotNone(torusPrim)
+        planePrim = shapeStage.GetPrimAtPath('/sample_usdShape/pPlane1')
+        self.assertIsNotNone(planePrim)
+
+        ufeSegment = mayaUtils.createUfePathSegment("|sample_usd|sample_usdShape")
+
+        #Sphere1
+        pSphere1UfePath = ufe.Path([ufeSegment, usdUtils.createUfePathSegment("/pSphere1")])
+        pSphere1UfeItem = ufe.Hierarchy.createItem(pSphere1UfePath)
+        self.assertIsNotNone(pSphere1UfeItem)
+
+        #Sphere2
+        pSphere2UfePath = ufe.Path([ufeSegment, usdUtils.createUfePathSegment("/pSphere2")])
+        pSphere2UfeItem = ufe.Hierarchy.createItem(pSphere2UfePath)
+        self.assertIsNotNone(pSphere2UfeItem)
+
+        #Torus
+        pTorusUfePath = ufe.Path([ufeSegment, usdUtils.createUfePathSegment("/pTorus1")])
+        pTorusUfeItem = ufe.Hierarchy.createItem(pTorusUfePath)
+        self.assertIsNotNone(pTorusUfeItem)
+
+        #Plane
+        pPlaneUfePath = ufe.Path([ufeSegment, usdUtils.createUfePathSegment("/pPlane1")])
+        pPlaneUfeItem = ufe.Hierarchy.createItem(pPlaneUfePath)
+        self.assertIsNotNone(pPlaneUfeItem)
```

Comment:
Is this the same as line 53?  I know this is a test, but you might want to factor this out.